Hermes/hermes.py

```diff
@@ -100,6 +111,19 @@ def poll():
   # The poller can move forward without waiting for a response here.
   helper.urlfetch_async(request)
 
+def cache_node_stats():
+  try:
+    STATS['node'] = helper.update_node_cache()
+  except (AppControllerException, InfrastructureManagerException) as e:
```

Comment:
@cdonati if I caught them in the helper I couldn't log anything about them, or that was the experience I had with it.